README.md

```diff
@@ -7,360 +7,419 @@
 
-## Campaign finance for everyone
+## About this project
 The Federal Election Commission (FEC) releases information to the public about money that’s raised and spent in federal elections — that’s elections for US president, Senate, and House of Representatives.
 
 Are you interested in seeing how much money a candidate raised? Or spent? How much debt they took on? Who contributed to their campaign? The FEC is the authoritative source for that information.
 
 betaFEC is a collaboration between [18F](http://18f.gsa.gov) and the FEC. It aims to make campaign finance information more accessible (and understandable) to all users.
 
-## FEC repositories
-We welcome you to explore, make suggestions, and contribute to our code.
 
-This repository, [openFEC](https://github.com/18F/openfec), is home to betaFEC’s API.
-
-### All repositories
+## This repository, [openFEC](https://github.com/18F/openfec), is home to betaFEC’s API
+All FEC repositories:
 - [FEC](https://github.com/18F/fec): a general discussion forum. We [compile feedback](https://github.com/18F/fec/issues) from betaFEC’s feedback widget here, and this is the best place to submit general feedback.
 - [openFEC](https://github.com/18F/openfec): betaFEC’s API
-- [swagger](https://github.com/18F/swagger-ui):this generates our interactive API documentation
-- [openFEC-web-app](https://github.com/18f/openfec-web-app): the betaFEC web app for exploring campaign finance data
-- [fec-style](https://github.com/18F/fec-style): shared styles and user interface components
-- [fec-cms](https://github.com/18F/fec-cms): the content management system (CMS) for betaFEC
-- [proxy](https://github.com/18F/fec-proxy): this is a light weight app that coordinates the paths between the web app and CMS
+- [swagger-ui](https://github.com/18F/swagger-ui): forked repo that generates our interactive API documentation
+- [openFEC-web-app](https://github.com/18f/openfec-web-app): the web app for exploring campaign finance data
+- [fec-style](https://github.com/18F/fec-style): shared styles and user interface components, including this project's glossary and feedback tools
+- [fec-cms](https://github.com/18F/fec-cms): this project's content management system (CMS) 
+- [proxy](https://github.com/18F/fec-proxy): this is a lightweight app that coordinates the paths between the web app and CMS
 
 
 ## Get involved
-We’re thrilled you want to get involved!
+We welcome you to explore, make suggestions, and contribute to our code. 
 - Read our [contributing guidelines](https://github.com/18F/openfec/blob/master/CONTRIBUTING.md). Then, [file an issue](https://github.com/18F/fec/issues) or submit a pull request.
-- [Send us an email](mailto:betafeedback@fec.gov).
-- If you’re a developer, follow the installation instructions in the README.md page of each repository to run the apps on your computer.
+- If you'd rather send us an email, [we're thrilled to hear from you](mailto:betafeedback@fec.gov)!
```

Comment:
Ack, I thought I removed all of the smart/smart-like quotes to make them plain text, good catch.